src/components/layout/RightSection.vue

```diff
@@ -134,8 +134,8 @@ export default {
     selectCurrency(selectedCurrency) {
       this.updateCurrency(selectedCurrency);
     },
-    roundAE(value) {
-      return new BigNumber(value).toFixed(2);
+    roundAE() {
```

Comment:
That would require also to remove the parameter from calls in the template https://github.com/aeternity/superhero-ui/pull/459/files#diff-c2c86662ffc089c65d251de49ee92ae6R33